test/examples/data_view_field_editor_example/data_view_field_editor_example.ts

```diff
@@ -6,19 +6,40 @@
  * Side Public License, v 1.
  */
 
+import expect from '@kbn/expect';
 import { PluginFunctionalProviderContext } from '../../plugin_functional/services';
 
 // eslint-disable-next-line import/no-default-export
 export default function ({ getService }: PluginFunctionalProviderContext) {
   const testSubjects = getService('testSubjects');
+  const find = getService('find');
 
   describe('', () => {
     it('finds a data view', async () => {
       await testSubjects.existOrFail('dataViewTitle');
     });
+
     it('opens the field editor', async () => {
       await testSubjects.click('addField');
       await testSubjects.existOrFail('flyoutTitle');
     });
+
+    it('uses preconfigured options for a new field', async () => {
+      // wonky way to check the checkbox in a test
```

Comment:
I hit a strange problem when the code simply put the `data-test-subj` on the EuiCheckbox element: using `testSubjects.setCheckbox()` wasn't finding the correct element to click